packages/components/ovh-shell/src/plugin/i18n/index.ts

```diff
@@ -3,11 +3,7 @@ import { KeyPairName } from '@ovh-ux/manager-config/types/locale';
 
 import Shell from '../../shell/shell';
 
-export interface I18nPlugin {
```

Comment:
why removing the plugin interface definition with an abstract Record<string, CallableFunction> ?

Reply:
Because it caused problems when integrating elsewhere, ideally, we should have Plugin definitions, with a `Plugin` interface, and other child interfaces that inherit from it. 

We should work on all Plugin definitions like we said, to have a more generic class etc